docs/GettingStarted.rst

```diff
@@ -1290,8 +1286,16 @@ The action is successfully done after confirming it as for the two options above
 
 You may display the home page to verify that the news is available in the appropriate block.
 
+You may display one of the following type of blocks :
+
+- Blog block: 
+
 |image172|
 
+- Latest News block: 
+
+|image198|
+
```

Comment:
Shouldn't we also explain here how to select the layout, and even how to add the portlet with the good parameters (folder, query, ...) ?

Reply:
I entend to do it in admin guide as users have no access to this (via administering eXo platform https://docs.exoplatform.org/en/latest/Administration.html#managingpages-editingpage )

Reply:
:+1: